beacon_chain/eth2_network.nim

```
@@ -890,8 +894,12 @@ proc init*(T: type Eth2Node, conf: BeaconNodeConf, enrForkId: ENRForkID,
   result.switch = switch
   result.wantedPeers = conf.maxPeers
   result.peerPool = newPeerPool[Peer, PeerID](maxPeers = conf.maxPeers)
-  result.connectTimeout = 1.minutes
-  result.seenThreshold = 5.minutes
+  when not defined(local_testnet):
+    result.connectTimeout = 1.minutes
+    result.seenThreshold = 5.minutes
+  else:
+    result.connectTimeout = 10.seconds
+    result.seenThreshold = 10.seconds
```

Won't this alter the testnet's behaviour beyond just making it fail faster?

---

Yes?